web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.test.tsx

```diff
@@ -136,7 +149,10 @@ test('starting the agent flips the workspace autoStart flag to true', async () =
     ),
   });
 
-  await act(() => result.current.startAgent());
+  await act(async () => {
+    const [, error] = await result.current.startAgent();
+    expect(error).toBeFalsy();
```

Comment:
Similar to the comment above, this test was failing without a mock, but the test itself was written in a way where the failure message was not helpful.

Checking the error of `startAgent` both makes the test more correct and also helps with debugging.